sdk/eventhub/azure-eventhub/azure/eventhub/_client_base.py

```diff
@@ -229,7 +229,12 @@ def _management_request(self, mgmt_msg, op_type):
                     status_code_field=b"status-code",
                     description_fields=b"status-description",
                 )
-                return response
+                status_code = response.application_properties[b"status-code"]
+                if status_code < 400:
```

Comment:
I would like to see a test case added for this scenario :)

Reply:
Don't know how to reproduce this issue in a certain way for now. It barely happens.